python/pyspark/ml/clustering.py

```diff
@@ -291,6 +292,284 @@ def _create_model(self, java_model):
         return BisectingKMeansModel(java_model)
 
 
+class LDAModel(JavaModel):
+    """ A clustering model derived from the LDA method.
+
+    Latent Dirichlet Allocation (LDA), a topic model designed for text documents.
+    Terminology
+    - "word" = "term": an element of the vocabulary
+    - "token": instance of a term appearing in a document
+    - "topic": multinomial distribution over words representing some concept
+    References:
+    - Original LDA paper (journal version):
+    Blei, Ng, and Jordan.  "Latent Dirichlet Allocation."  JMLR, 2003.
+
+    .. versionadded:: 2.0.0
+    """
+
+    @since("2.0.0")
+    def isDistributed(self):
+        """Indicates whether this instance is of type DistributedLDAModel"""
+        return self._call_java("isDistributed")
+
+    @since("2.0.0")
+    def vocabSize(self):
+        """Vocabulary size (number of terms or terms in the vocabulary)"""
+        return self._call_java("vocabSize")
+
+    @since("2.0.0")
+    def topicsMatrix(self):
+        """Inferred topics, where each topic is represented by a distribution over terms."""
+        return self._call_java("topicsMatrix")
```

Comment:
The Python API doc should consistent with Scala one, should also include the ```WARNING``` to tell users that this API may cause collecting a large amount of data to the driver.